Server/Ebenezer/User.h

```diff
@@ -282,6 +282,7 @@ class CUser : public CIOCPSocket2
 	void Home();
 	void ReportBug(char* pBuf);
 	int GetEmptySlot(int itemid, int bCountable);
+	int GetNumberOfEmptySlots();
```

Comment:
Mark this as const (at the end), so it knows that we're not actually modifying class members here